### PR TITLE
Create a Cron job to deactivate quotes which belong to created orders

### PR DESCRIPTION
**Pull request by johnmccombs1**
# Description
Create a Cron job to deactivate quotes which belong to created orders

Fixes: https://app.asana.com/0/564264490825835/1157946616328017

# Type of change

- [x] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [x] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# Checklist:

- [x] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] New and existing unit tests pass locally with my changes.
- [ ] I have created or modified unit tests to sufficiently cover my changes.
- [x] I have added my Asana task link and provided a changelog message if applicable.

### CODE REVIEW

File: etc/crontab.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Cron:etc/crontab.xsd">
+    <group id="default">
+        <job name="deactivate_quote" instance="Bolt\Boltpay\Cron\DeactivateQuote" method="execute">
+            <schedule>0 2 * * *</schedule>
```
Comment (danac-gs) [inline etc/crontab.xml]: Once a day at 2 am. Not sure this is frequent enough. I'd set it once every hour.

File: etc/crontab.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Cron:etc/crontab.xsd">
+    <group id="default">
```
Comment (YevhenShBolt) [inline etc/crontab.xml]: cron_groups
I recommends to create a Bolt group, in which a Bolt team or merchant can setup time to run jobs regarding their needs.

File: Cron/DeactivateQuote.php [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2019 Bolt Financial, Inc (https://www.bolt.com)
```
Comment (YevhenShBolt) [inline Cron/DeactivateQuote.php]: I guess Bolt copyright for new files should be a 2020 for current year. 
@matt-thomason Is not it?

**Comment by danac-gs** [comment]
@johnmccombs1 @matt-thomason @njfaries @YevhenShBolt
With the solution in https://github.com/BoltApp/bolt-magento2/pull/523 this seems unnecessary.

**Pull request merged**